manifests/pnp4nagios/config.pp

```diff
@@ -0,0 +1,77 @@
+class icinga2::pnp4nagios::config inherits icinga2::pnp4nagios::params {
+
+  $monitoring_type = 'icinga'
+  $log_type = 'syslog'
+  $debug_lvl = 1
+  $system_date = generate('/usr/bin/date', '+%Z')
```

Comment:
If they should not be changed they shouldn't be variables. I think they should be changeable though, so just make them parameters.